config/enabled.yml

```diff
@@ -450,6 +450,11 @@ Style/MultilineIfThen:
   StyleGuide: '#no-then'
   Enabled: true
 
+Style/MultilineIfModifier:
+  Description: 'Ensures if/unless modifiers are only applied to single lines.'
```

Comment:
Generally, these descriptions are written to read as intructions on how to write more correct code. I'd suggest something like `Only use if/unless modifiers on single line statements`.